### PR TITLE
android: UI for pending contact connections, ios: translations, show profile picture in contact requests

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -132,6 +145,13 @@
     <string name="accept_contact_button">Принять</string>
     <string name="reject_contact_button">Отклонить</string>
 
+    <!-- Pending contact connection alert dialogues -->
+    <string name="you_invited_your_contact">Вы пригласили ваш контакт</string>
+    <string name="you_accepted_connection">Вы приняли приглашение соединениться</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:150]:
```suggestion
    <string name="you_accepted_connection">Вы приняли приглашение соединиться</string>
```

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -24,6 +24,15 @@
     <string name="unknown_message_format">неизвестный формат сообщения</string>
     <string name="invalid_message_format">неверный формат сообщения</string>
 
+    <!-- PendingContactConnection - ChatModel.kt -->
+    <string name="connection_local_display_name">connection <xliff:g id="connection ID" example="1">%1$d</xliff:g></string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:28]:
```suggestion
    <string name="connection_local_display_name">соединение <xliff:g id="connection ID" example="1">%1$d</xliff:g></string>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 3]
```diff
@@ -690,6 +710,11 @@ To connect, please ask your contact to create another connection link and check
         <target>Вы</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="You accepted connection" xml:space="preserve">
+        <source>You accepted connection</source>
+        <target>Вы приняли приглашение соединениться</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:715]:
```suggestion
        <target>Вы приняли приглашение соединиться</target>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 4]
```diff
@@ -829,11 +859,31 @@ SimpleX серверы не могут получить доступ к ваше
         <target>соединитесь с разработчиками.</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="connecting…" xml:space="preserve">
+        <source>connecting…</source>
+        <target>соединеняется…</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:864]:
```suggestion
        <target>соединяется…</target>
```

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 5]
```diff
@@ -24,6 +24,15 @@
...
+    <string name="display_name_connection_established">соединение установлено</string>
+    <string name="display_name_invited_to_connect">приглашение соединиться</string>
+    <string name="display_name_connecting">соединеняется…</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:31]:
```suggestion
    <string name="display_name_connecting">соединяется…</string>
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 6]
```diff
@@ -151,6 +151,9 @@
 /* No comment provided by engineer. */
 "Connecting server… (error: %@)" = "Устанавливается соединение с сервером… (ошибка: %@)";
 
+/* chat list item title */
+"connecting…" = "соединеняется…";
```
Comment (jr-simplex) [inline apps/ios/ru.lproj/Localizable.strings:155]:
```suggestion
"connecting…" = "соединяется…";
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 7]
```diff
@@ -829,11 +859,31 @@ SimpleX серверы не могут получить доступ к ваше
...
+        <note>chat list item title</note>
+      </trans-unit>
+      <trans-unit id="connection established" xml:space="preserve">
+        <source>connection established</source>
+        <target>соединение установлено</target>
+        <note>chat list item title (it should not be shown</note>
+      </trans-unit>
+      <trans-unit id="connection:%@" xml:space="preserve">
+        <source>connection:%@</source>
+        <target>connection:%@</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:874]:
```suggestion
        <target>соединение:%@</target>
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 8]
```diff
@@ -169,6 +175,9 @@
 /* No comment provided by engineer. */
 "Connection timeout" = "Превышено время соединения";
 
+/* connection information */
+"connection:%@" = "connection:%@";
```
Comment (jr-simplex) [inline apps/ios/ru.lproj/Localizable.strings:179]:
```suggestion
"connection:%@" = "соединение:%@";
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 9]
```diff
@@ -454,6 +484,9 @@
 /* No comment provided by engineer. */
 "You" = "Вы";
 
+/* No comment provided by engineer. */
+"You accepted connection" = "Вы приняли приглашение соединениться";
```
Comment (jr-simplex) [inline apps/ios/ru.lproj/Localizable.strings:488]:
```suggestion
"You accepted connection" = "Вы приняли приглашение соединиться";
```

**Comment by jr-simplex** [comment]
a couple more minor fixes I've noticed when compiling that I couldn't suggest in this PR
https://github.com/simplex-chat/simplex-chat/pull/574/files
(it doesn't include Localizable.strings files since they would be recompiled with changes in this PR anyway)

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff (hunk 7 above)
Comment (epoberezkin) [reply apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:874]: don't want to translate it, it's not in UI

**Pull request merged**